engines/nfs.c

```diff
@@ -147,12 +151,12 @@ static int nfs_event_loop(struct thread_data *td, bool flush) {
 		DEBUG_PRINT("why bother me?\n");
 		return o->buffered_event_count;
 	}
-	
+
 
 #define SHOULD_WAIT() (o->outstanding_events == td->o.iodepth || (flush && o->outstanding_events))
-	
+
 	do {
-		int timeout = SHOULD_WAIT() ? -1 : 0;
+		int timeout = SHOULD_WAIT() ? 60000 : 0;
```

Comment:
this is should be a timeout we can pass into fio.

Reply:
We should default to duration of run if we can get that out of fio

Reply:
I was able to get the runtime from fio and use that to set the timeout